assets/prometheus-k8s/rules.yaml

```diff
@@ -611,6 +611,23 @@ spec:
       record: count:up1
     - expr: count without(instance, pod, node) (up == 0)
       record: count:up0
+  - name: openshift-general.rules
+    rules:
+    - alert: TargetDown
+      annotations:
+        description: '{{ printf "%.4g" $value }}% of the {{ $labels.job }}/{{ $labels.service
```

Comment:
I don't see a problem with this description going upstream as well, any objections to that?

Reply:
Because upstreams probably can’t assume “contacting suppprt”?